### DISCUSSION TITLE
How can I save an empty bundle?

### DISCUSSION THREAD

**Discussion by ofseed** [stevearc/overseer.nvim#80] [Q&A] [answered]
I am using overseer with a session manager and found that the function
`save_task_bundle` only actually saves tasks when there are tasks.

Therefore, if I dispose all the tasks and exit, execute `save_task_bundle` automatically which does nothing.
Next time I load this session again, it will load the tasks I already disposed of.

I want to save an empty bundle when I disposed of all tasks
so that these tasks will not load again when I load this session,
what should I do?

**Comment by stevearc** [comment, marked as answer]
This was an oversight in the bundle saving logic. I've pushed up a change; it should work as expected now
